transport/src/test/java/io/zeebe/transport/impl/AtomixTransportTest.java

```diff
@@ -182,7 +182,7 @@ public void shouldRetryOnInvalidResponse() throws Exception {
         nodeAddressSupplier, (response) -> false, new Request("messageABC"), Duration.ofSeconds(3));
 
     // then
-    final var success = retryLatch.await(1, TimeUnit.SECONDS);
+    final var success = retryLatch.await(2, TimeUnit.SECONDS);
```

Comment:
I guess this is unintended to be in this PR?

Reply:
This is intentional. After activating test coverage, this test failed much more often than before.